osi_object.proto

```diff
@@ -835,67 +837,143 @@ message MovingObject
             //
             // \image html OSI_TYPE_DELIVERY_VAN.svg
             //
+            // \note Deprecated differentiation, use TYPE_VAN instead
+            //
             TYPE_DELIVERY_VAN = 6;
 
-            // Vehicle is a (heavy) truck.
+            // A van is a motorized vehicle with a larger cargo area than a car,
+            // used for transporting goods or people.
+            //
+            TYPE_VAN = 6;
```

Comment:
Using the same index twice works as an alias, right?

Reply:
Yes, but it requires `option allow_alias = true;` to be set. This was already added in 3.7.0 for the vehicle types because the alias TYPE_CAR (index 4) was added.